src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -136,12 +136,12 @@ public static string GetValueForConfigKey(ISettings settings, string key, bool s
         /// <summary>
         /// Logs each section of the configuration settings that will be applied, grouped by file path.
         /// </summary>
-        public static void LogSectionsWithPaths(Dictionary<string, VirtualSettingSection> sections, ILogger logger)
+        public static void LogSectionsWithPaths(IEnumerable<string> sections, Settings settings, ILogger logger)
```

Comment:
`LogSectionsWithPaths` and `LogSectionsNoPaths` have a lot of the same code. Can we deduplicate that code into a helper method they both use or the same method with a usePaths bool parameter?